### PR TITLE
Wrap Process sequence

### PR DESCRIPTION
**Pull request by sofianehaddad**
This fixes http://trac.openturns.org/ticket/842

### CODE REVIEW

**Comment by jschueller** [comment]
+1

**Comment by regislebrun** [comment]
+1

**Comment by jschueller** [comment]
there's a conflict in the changelog I guess

**Comment by sofianehaddad** [comment]
@jschueller yes fixed

**Comment by jschueller** [comment]
The docstring mention ProcessCollection, you might want to change that to "sequence of" ..,
also maybe the template in UncertaintyProcessTemplateDefs.i can be removed too:
```
$ grep -nr ProcessCollection python/src/
python/src/UncertaintyProcessTemplateDefs.i:2:%template(ProcessCollection) OT::Collection<OT::Process>;
python/src/AggregatedProcess.i:10:%template(ProcessCollection) OT::Collection<OT::Process>;
python/src/AggregatedProcess_doc.i.in~:9:collProc :  :class:`~openturns.ProcessCollection`
python/src/AggregatedProcess_doc.i.in~:79:%feature("docstring") OT::AggregatedProcess::setProcessCollection
python/src/AggregatedProcess_doc.i.in:9:collProc :  :class:`~openturns.ProcessCollection`
python/src/AggregatedProcess_doc.i.in:29:>>> myProcColl = ot.ProcessCollection()
python/src/AggregatedProcess_doc.i.in:82:%feature("docstring") OT::AggregatedProcess::setProcessCollection
python/src/AggregatedProcess_doc.i.in:87:collProc : :class:`~openturns.ProcessCollection`
```

**Comment by jschueller** [comment]
Same thing for setProcessCollection:
```
%feature("docstring") OT::AggregatedProcess::setProcessCollection
"Set the collection of processes.

Parameters
----------
collProc : :class:`~openturns.ProcessCollection`
    Collection of processes which all have the same spatial dimension.
```

**Comment by sofianehaddad** [comment]
@jschueller fixed. But it is strange that AggregatedProcess::setProcessCollection is public.

**Comment by jschueller** [comment]
indeed, it does not seems very useful, but this is another PR as we have other objects that expose such apis eg ComposedDistribution::setDistributionCollection

**Comment by jschueller** [comment]
+1

File: python/src/AggregatedProcess_doc.i.in [hunk 1]
```diff
@@ -84,6 +81,17 @@ For example, if :math:`X_0=(X_0^0, X_0^1)`, :math:`X_1=(X_1^0, X_1^1)` and :math
...
+"
+
+// ---------------------------------------------------------------------
+
+%feature("docstring") OT::AggregatedProcess::getProcessCollection
+"Get the collection of processes.
+
+Returns
+-------
+collProc : sequence of :class:`~openturns.Process`
```
Comment (jschueller) [inline python/src/AggregatedProcess_doc.i.in]: actually the return type is ProcessCollection, not a generic sequence

**Comment by sofianehaddad** [comment]
@jschueller you are right, thanks. It is fixed

**Comment by jschueller** [comment]
+1

**Comment by dbarbier** [comment]
+1

**Pull request merged**